wasswa-derick/NairobiJavaGeeks
# [Feature #156187450] Add Network utility class

#### What does this PR do?
* Add unit tests

#### Description of Task to be completed?
*  Check the device's network connectivity when the user pings the server without an internet connection.
* Update the README doc with the correct information about the project.

#### How should this be manually tested?
* Clone the project, ```git clone https://github.com/wasswa-derick/NairobiJavaGeeks.git```.
* In your terminal, ```cd NairobiJavaGeeks```
* Checkout to the ```ft-network-utility-156187450``` branch.
* Check for this branch ```ft-network-utility-156187450``` on the repository and read through its readme document.
* Run the application on your device with internet connection disabled. The application should alert the user about this status and allow the user an action to activate the internet.


#### What are the relevant pivotal tracker stories?
[#156185411](https://www.pivotaltracker.com/story/show/156185411)
[#156187450](https://www.pivotaltracker.com/story/show/156187450)

#### ScreenShots

> Updated ReadMe

<img width="1061" alt="screen shot 2018-10-04 at 18 22 18" src="https://user-images.githubusercontent.com/39955231/46484475-87490700-c802-11e8-873f-1e0c26fa36a1.png">

> Network Utility 

![screenshot_2018-10-04-17-58-23](https://user-images.githubusercontent.com/39955231/46484782-1eae5a00-c803-11e8-990f-920a4ce3dc6c.png)

> Splash Screen

![screenshot_2018-10-05-11-35-07](https://user-images.githubusercontent.com/39955231/46524921-ebb3a700-c892-11e8-8605-7e3e7325a552.png)
